maps_dashboards/public/components/layer_config/documents_config/document_layer_source.tsx

```diff
@@ -136,6 +136,7 @@ export const DocumentLayerSource = ({ setSelectedLayerConfig, selectedLayerConfi
                 setIndexPattern(newIndexPattern);
               }}
               isClearable={false}
+              isInvalid={indexPattern instanceof IndexPattern == false}
```

Comment:
```suggestion
              isInvalid={!(indexPattern instanceof IndexPattern)}
```

Reply:
Yeah. Wanted to avoid parenthesis and ! . Is this not acceptable?

Reply:
The suggested code simplified the logic, yeah, either way works! But here we need use `===` to comparing two variables in TS/JS.